config/default.yml

```diff
@@ -130,6 +130,11 @@ Rails/ActiveRecordAliases:
   VersionAdded: '0.53'
   SafeAutoCorrect: false
 
+Rails/ActiveRecordCalculation:
+  Description: 'Use ActiveRecord calculation methods instead of `pluck` followed by Enumerable methods.'
+  Enabled: pending
+  VersionAdded: '<<next>>'
```

Comment:
This would need to be marked unsafe because Ruby's sum/max aren't always compatible with the DB's:

```ruby
User.pluck(:name).sum # "AliceBobCathy"
User.sum(:name) # PG::UndefinedFunction: ERROR:  function sum(character varying) does not exist
```